client/homebrew/pages/sharePage/sharePage.jsx

```diff
@@ -103,8 +88,13 @@ const SharePage = createClass({
 				</Nav.section>
 
 				<Nav.section>
-					{this.renderPrint()}
-					{this.renderSourceDropdown()}
+					{this.props.shareId && <PrintLink shareId={this.processShareId()} />
+					<Nav.item icon='fas fa-code' color='red' className='source'
+						onMouseEnter={()=>this.handleDropdown(true)}
+						onMouseLeave={()=>this.handleDropdown(false)}>
+						source
+						{this.renderDropdown()}
+					</Nav.item>}
```

Comment:
Similarly, we can take the closing `}` and move it to a new line so it's more legible which components are enclosed by this check.

```
  </Nav.item>
}
```

I think Lint might actually force this anyway.

Reply:
I should be able to run it through the Linter shortly in the next hour or so.